packages/swapper/src/swappers/CowSwapper/endpoints.ts

```diff
@@ -78,6 +79,8 @@ export const cowApi: SwapperApi = {
       ),
     } = tradeQuote
 
+    if (receiveAddress === zeroAddress) throw new Error('No receive address found in quote')
```

Comment:
we don't return `zeroAddress` on the cowswap quote. can use `isExecutableTrade` for safety.

Reply:
Very nice catch! a7007bab60